clang/test/CodeGenSYCL/kernel-caller-entry-point.cpp

```diff
@@ -10,9 +10,7 @@
 // RUN: %clang_cc1 -fsycl-is-device -emit-llvm -aux-triple x86_64-pc-windows-msvc -triple amdgcn-amd-amdhsa -std=c++17 %s -o - | FileCheck --check-prefixes=CHECK-DEVICE,CHECK-AMDGCN %s
 // RUN: %clang_cc1 -fsycl-is-device -emit-llvm -aux-triple x86_64-pc-windows-msvc -triple nvptx-nvidia-cuda -std=c++17 %s -o - | FileCheck --check-prefixes=CHECK-DEVICE,CHECK-NVPTX %s
 // RUN: %clang_cc1 -fsycl-is-device -emit-llvm -aux-triple x86_64-pc-windows-msvc -triple nvptx64-nvidia-cuda -std=c++17 %s -o - | FileCheck --check-prefixes=CHECK-DEVICE,CHECK-NVPTX %s
-// RUN: %clang_cc1 -fsycl-is-device -emit-llvm -aux-triple x86_64-pc-windows-msvc -triple spir-unknown-unknown -std=c++17 %s -o - | FileCheck --check-prefixes=CHECK-DEVICE,CHECK-SPIR %s
```

Comment:
I think it is better to disable with a comment pointing to a gh issue instead of simply removing the lines.

Reply:
Thanks @Fznamznon , I agree that makes sense. I filed the following issue to clarify what host/device target combinations are intended to work and to improve diagnostics for invalid/unsupported combinations (it is pretty messy/yucky right now). Once there is clarity regarding what combinations are expected to work (and therefore should be tested), I'll update the test upstream so that pulldown coordinators won't be impacted.
- https://github.com/intel/llvm/issues/18304: Improper handling of presumably invalid host and device target combinations